scripts/data_analysis/constants/common.py

```diff
@@ -1,8 +1,18 @@
 import pandas as pd
 import requests
 import yaml
+import os
 
 URL = 'https://api-dev.prison.service.justice.gov.uk/v3/api-docs'
+SCHEMA_FIELD_FILE = "outputs/Schema_field.csv"
+SCHEMA_PARENT_CHILD_FILE = "outputs/Schema_parent_child.csv"
+SCHEMA_DIAGRAM = "outputs/schema_hierachy.dot"
```

Comment:
Should all the naming of the files be consistent? Or is this on purpose to have the CSV files begin with a capital and the .dot file different?

Reply:
No you're right, they should be all consistent. Will change.